docs/contributor-guide/developer-guide/dev-testing.md

```diff
@@ -25,4 +25,16 @@ To view the output nicer you can use the [JSON Formatter](https://chrome.google.
 
 ## File Upload
 
-TBD
+Make sure to add `localstack` and `azurite` to `/etc/hosts` prior to trying file upload within the dev stack.
```

Comment:
can you make a small code block that people can copy and paste into /etc/hosts?

Reply:
Done. I ended up linking to the section of the docs that @jeanniefu already helpfully wrote on this topic!